src/sage/misc/latex.py

```diff
@@ -605,8 +520,8 @@ def __init__(self, bb=False, delimiters=["(", ")"],
         self._option["matrix_column_alignment"] = matrix_column_alignment
         self._option["macros"] = ""
         self._option["preamble"] = ""
-        self._option["engine"] = "pdflatex"
-        self._option["engine_name"] = "LaTeX"
+        self._option["engine"] = default_engine()[0]
```

Comment:
does this get run on each startup of Sage?

Reply:
Yes. Problem?

Reply:
But as usual, the checks will not run until requested. That is, it won't delay the startup time.

Reply:
Hmm. It does. I will find a fix.

Reply:
Now done.

Reply:
Would it make a sense that `sage_docbuild/conf.py` also uses the default engine detected in `sage/misc/latex.py`?

Reply:
only if it detects xelatex or lualatex,  otherwise we can't build docs.

I would say this detection should be delegated to ./configure

Reply:
OK. Then I leave it as a later work to one who knows `./configure`.

Reply:
Or no work. Perhaps it is not worth while to detect xelatex, as we are recommending to install lualatex for doc.

Reply:
Right, for sagemath_doc_pdf, it's declared as a dependency; so nothing to do.